cypress/support/accessibility.js

```diff
@@ -0,0 +1,54 @@
+export const checkA11y = options => {
```

Comment:
I find this a bit confusing. What is this local `checkA11y` and how does it differ from `cy.checkA11y`? Perhaps a comment would clarify.

It seems that this is only used once, below on line 43. Perhaps it could be inlined instead of defining a single-use function?

Also, it is exported, but apparently not being used outside this file (in a meaningful way), so exporting seems unnecessary.